services/ui-src/src/components/pages/Help/HelpPage.tsx

```diff
@@ -25,9 +25,11 @@ export const HelpPage = () => {
           cardprops={sx.card}
         />
       </Box>
-      <Box sx={sx.faqAccordionBox}>
-        <FaqAccordion accordionItems={accordionItems} />
-      </Box>
+      {accordionItems.length > 1 && (
```

Comment:
Request: As-is, an accordion with only 1 item would not show, because the length is only equal to 1 and not greater than 1. What you probably want is `accordionItems.length`, which will evaluate to `true` if it is anything greater than 0.
```suggestion
      {accordionItems.length && (
```

Reply:
Note: If we wanted to delete the `accordionItems` array from the verbiage file entirely right now we could do that too.